docs/source/tutorials/tutorial_tensordot_with_tc.rst

```diff
@@ -0,0 +1,117 @@
+Using TC to get fast CUDA code for Tensor Contraction
+=====================================================
+
+In this tutorial, we will see how we can start from a random math operation,
+express it in TC language and easily get the fast CUDA code for it. We will also
+see how to tune the CUDA code to a better performance. All of this is possible with
+only 3-4 lines of code. Let's get started.
+
+For this tutorial, you will need to install Tensor Comprehensions binary. You can
+get binary builds of Tensor Comprehensions with ``conda install -y -c pytorch -c prigoyal tensor_comprehensions``.
+
+About TensorDot operation
+-------------------------
+
+First, we find an operation that we want to generate fast CUDA code for. A lot of
+operations like convolution, pooling are standard and have CUDA code easily available, so
+rather we are going to pick a new and different operation. How do we find a new operation?
+
+**Sources**: Maybe there is a research paper idea you have like KRU or there is a
```

Comment:
Should we mention KRU publicly

Reply:
I think this particular should be okay, we are not providing any TC for KRU or any example but simply saying that there is the research idea, go try